test_conformance/math_brute_force/unary_two_results_float.cpp

```diff
@@ -258,12 +260,6 @@ int TestFunc_Float2_Float(const Func *f, MTdata d, bool relaxedMode)
             }
         }
 
-        if (gSkipCorrectnessTesting)
-        {
-            if (isFract && gIsInRTZMode) (void)set_round(oldRoundMode, kfloat);
```

Comment:
There's an extra `if` statement getting removed here and in `unary_two_results_half.cpp` compared to the other tests, is that accounted for by this change?

Reply:
Makes sense, thanks :)